docs/docs/spec.md

```diff
@@ -968,33 +971,19 @@ The process of applying a [nullness operator] requires two inputs:
 
 The output of the process is an augmented type.
 
-The process is as follows:
-
-First, based on the pair of nullness operators (the one to apply and the one
-from the augmented type), compute a "desired nullness operator." Do so by
-applying the following rules in order. Once one condition is met, skip the
-remaining conditions.
-
--   If the nullness operator to apply is `MINUS_NULL`, the desired nullness
-    operator is `MINUS_NULL`.
--   If either nullness operator is `UNION_NULL`, the desired nullness operator
-    is `UNION_NULL`.
--   If either nullness operator is `UNSPECIFIED`, the desired nullness operator
-    is `UNSPECIFIED`.
--   The desired nullness operator is `NO_CHANGE`.
-
-Then, if the input augmented type is *not* an [intersection type], the output is
-the same as the input but with its nullness operator replaced with the desired
-nullness operator.
-
-Otherwise, the output is an intersection type. For every element `Tᵢ` of the
-input type, the output type has an element that is the output of applying the
-desired nullness operator to `Tᵢ`.
-
-> In this case, the desired nullness operator is always equal to the nullness
-> operator to apply that was an input to this process. That's because the
-> nullness operator of the intersection type itself is defined to always be
-> `NO_CHANGE`.
+To determine the output, apply the following rules in order. Once one condition
+is met, skip the remaining conditions.
+
+-   If the nullness operator to apply is `NO_CHANGE`, then the output augmented
+    type is the input augmented type.
+
+-   If the input augmented type is an [intersection type], then the output is
+    also an intersection type. For every element `Tᵢ` of the input type, the
+    output type has an element that is the output of applying the desired
+    nullness operator to `Tᵢ`.
+
+-   The output is a type that is the same as the input augmented type except
+    with its nullness operator set to the nullness operator to apply.
```

Comment:
```suggestion
To determine the output, apply the following rules in order.

-   If the nullness operator to apply is `NO_CHANGE`, then the output augmented
    type is the input augmented type.

-   Otherwise, if the input augmented type is an [intersection type], then the output is
    also an intersection type. For every element `Tᵢ` of the input type, the
    output type has an element that is the output of applying the desired
    nullness operator to `Tᵢ`.

-   Otherwise, the output is a type that is the same as the input augmented type except
    with its nullness operator set to the nullness operator to apply.
```

Is this more direct?

Reply:
Probably. We have a couple other "skip the remaining conditions" sections that you're welcome to touch up if you'd like.